bp_common/test/Makefile.frag

```diff
@@ -160,7 +177,27 @@ RV64_V_TESTS = \
   rv64ui-v-subw    \
   rv64ui-v-sw      \
   rv64ui-v-xor     \
-  rv64ui-v-xori
+  rv64ui-v-xori    \
+  \
+  #rv64ua-v-amoadd_d  \
```

Comment:
Why are these tests commented out?

Reply:
Oh right, I forgot.  A couple of them fail in spike, probably worth looking into since they're VM+emulation tests